tests/software/bcftools/test.yml

```diff
@@ -0,0 +1,81 @@
+- name: Run bcftools filter test workflow
```

Comment:
All of these tests will need to go in their own files now so we can modularise the tests too e.g. https://github.com/nf-core/modules/blob/master/tests/software/samtools/flagstat/test.yml